src/orders/views/OrderDraftList/OrderDraftList.tsx

```diff
@@ -233,9 +250,9 @@ export const OrderDraftList: React.FC<OrderDraftListProps> = ({ params }) => {
             defaultMessage="{counter,plural,one{Are you sure you want to delete this order draft?} other{Are you sure you want to delete {displayQuantity} order drafts?}}"
             description="dialog content"
             values={{
-              counter: maybe(() => params.ids.length),
+              counter: maybe(() => selectedRowIds.length),
               displayQuantity: (
-                <strong>{maybe(() => params.ids.length)}</strong>
+                <strong>{maybe(() => selectedRowIds.length)}</strong>
```

Comment:
**suggestion**: so MAYBE :D you can get rid of that and use proper TS? (if you touch this code)